tests/test_client.py

```diff
@@ -158,6 +163,53 @@ def test_get_raise_empty(self, client):
         with pytest.raises(ValueError):
             client.get("e3d82cda-4737-4af9-8d17-d9dfda8703d0", raise_empty=True)
 
+    def test_get_keyerror(self, client):
+        series_id = "test_series_id"
+        start = "2023-01-01"
+        end = "2023-01-02"
+
+        response_mock = MagicMock()
+        response_mock.status_code = 200
+        response_mock.json.return_value = {
+            "Files": [
+                {"Chunks": "file1"},
+                {"Chunks": "file2"},
+            ]  # mock files with correct structure
+        }
+
+        client._auth_session.get = MagicMock(return_value=response_mock)
+
+        def mock_storage_get(blob_sequence_i):
+            if blob_sequence_i == "file1":
+                return pd.DataFrame(
+                    {
+                        "index": [1672358410000000000, 1672358400000000000],
+                        "values": [100, 200],
+                    }
+                )
+            elif blob_sequence_i == "file2":
+                return pd.DataFrame(
+                    {
+                        "index": [1672358410000000000, 1672358420000000000],
+                        "values": [200, 400],
+                    }
+                )
+            else:
+                raise ValueError("Unexpected blob_sequence_i value")
+
+        client._storage.get = MagicMock(side_effect=mock_storage_get)
+
+        with patch(
+            "datareservoirio.client._blob_sequence_days",
+            side_effect=_mock_blob_sequence_days,
+        ):
+            with patch("datareservoirio.logging.warning") as mock_logging_warning:
+                result = client.get(series_id, start, end)
+                mock_logging_warning.assert_called_once_with(
+                    "KeyError. The data will be sorted to attempt to resolve the issue. Please note that this operation may take some time."
+                )
+                assert isinstance(result, pd.Series)
+
```

Comment:
This kind of test does not follow the same pattern as the other tests. This is because I think the setup is a little hard, but can change it, but then I need some more time. What do you think @branislav-jenco-4ss

Reply:
I _think_ this looks good :)